### PR TITLE
Implemented Flipper flag

### PR DESCRIPTION
**Pull request by tahir-khalid**
## Description
 
Implemented Flipper flag to enable and disable features related to the epic PSD-3041.

This flag allows enabling and disabling of features (see PSD-4383 for implementation instructions).

### CODE REVIEW

File: db/migrate/20241220122808_submit_notification_reminder.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class SubmitNotificationReminder < ActiveRecord::Migration[7.1]
+  def up
+    Flipper.enable(:submit_notification_reminder)
+    # usage:
```
Comment (meeravnagesh) [inline db/migrate/20241220122808_submit_notification_reminder.rb:4]: please remove these comments and add it as a comment in the ticket

File: db/migrate/20241220122808_submit_notification_reminder.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    #       if Flipper.enabled?(:submit_notification_reminder)
+    #          your_feature_code
+    #        else
+    #          existing_code
+    #        end
+  end
+
+  def down
+    Flipper.disable(:submit_notification_reminder)
+    # Flipper.remove(:submit_notification_reminder)
```
Comment (meeravnagesh) [inline db/migrate/20241220122808_submit_notification_reminder.rb:14]: same , remove the comments

File: db/migrate/20241220122808_submit_notification_reminder.rb (hunk 1 above)
Comment (tahir-khalid) [reply db/migrate/20241220122808_submit_notification_reminder.rb]: Removed.

File: db/migrate/20241220122808_submit_notification_reminder.rb (hunk 2 above)
Comment (tahir-khalid) [reply db/migrate/20241220122808_submit_notification_reminder.rb]: Removed (PSD-4383 has captured the details).

**Review by meeravnagesh — APPROVED** [review]

**Pull request merged**